frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -205,9 +205,9 @@ function roleSelected(evt: any) {
                 <form id="grantAccessForm" class="form-container">
                     <div class="row">
                         <div class="form-group col-md-3 px-0">
-                            <label for="domainInput" class="control-label"
```

Comment:
I can't find where the class "**control-label**" came form.
Perhaps this is old Bootstrap 3 class, not sure if for Bootstrap 5 this is "**[form-control](https://getbootstrap.com/docs/5.0/forms/form-control/)**".

Reply:
If that "control-label" does not do anything, could you do a quick search and remove them (or replace them with "[form-control](https://getbootstrap.com/docs/5.0/forms/form-control/)" from Bootstrap 5) from our FAM frontend? Please run and verify after change. Thank you Jota.

Reply:
I believe Catherine already took care of that

Reply:
No, failed to check missed one, see line 44 on github:
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/27d6138a-a1ab-49cb-8791-44979a3beaa7)

Reply:
oh, I'll gladly take care of that.

Reply:
I think we can remove all the class here `class="control-label label px-0"`, please verify remove these won't impact anything, thanks!

Reply:
removed